docs/uart.rst

```diff
@@ -0,0 +1,65 @@
+UART
+****
+
+.. py:module:: microbit.uart
+
+The ``uart`` module lets you talk to a device connected to your board using
+a serial interface.
+
+
+Functions
+=========
+
+.. py:function:: init(baudrate=9600, bits=8, parity=None, stop=1, pins=None)
+
+    Initialize serial communication with the specified parameters on the
+    specified ``pins``. Note that for correct communication, the parameters
+    have to be the same on both communicating devices.
+
+    The ``baudrate`` defines the speed of communication. Common baud
+    rates include:
+
+        * 9600
+        * 14400
+        * 19200
+        * 28800
+        * 38400
+        * 57600
+        * 115200
+
+    The ``bits`` defines the size of bytes being transmitted, and the board only
+    supports 8. The ``parity`` parameter defines how partity is checked, and it can
+    be ``None``, ``microbit.uart.ODD`` or ``microbit.uart.EVEN``. The ``stop``
+    parameter tells the number of stop bits, and has to be 1 for this board.
+
+    If no ``pins`` are specified, ``microbit.pin0`` is used as the TX pin, and
+    ``microbit.pin1`` as the RX pin. You can also specify which pins you want
+    by passing a tuple of two pins as ``pins``, the first one being TX, and the
+    second one, RX.
+
+    .. note::
+
+        When connecting the device, make sure you "cross" the wires -- the TX
+        pin on your board needs to be connected with the RX pin on the device,
+        and the RX pin -- with the TX pin on the device. Also make sure the
+        ground pins of both devices are connected.
+
+.. py:function:: any()
+
+    Return ``True`` if there are any bytes to be read in the input buffer.
+    Return ``False`` when the buffer is empty.
+
+.. py:function:: read(buf_in, size)
```

Comment:
All of those functions need proper descriptions and proper parameters. Unfortunately I don't know the Micropython C API well enough to guess them from the source code, and there is no other source of information.